api/prisma/seed-helpers/household-member-factory.ts

```diff
@@ -1,45 +1,54 @@
 import { Prisma, YesNoEnum } from '@prisma/client';
+import { randomInt } from 'crypto';
+import { HouseholdMemberRelationship } from '../../src/enums/applications/household-member-relationship-enum';
 import { addressFactory } from './address-factory';
-import { randomAdjective, randomNoun } from './word-generator';
+import { randomNoun } from './word-generator';
 import {
   randomBirthDay,
   randomBirthMonth,
   randomBirthYear,
 } from './number-generator';
 import { randomBoolean } from './boolean-generator';
 
-export const householdMemberFactorySingle =
-  (): Prisma.HouseholdMemberCreateWithoutApplicationsInput => {
-    const firstName = randomNoun();
-    const lastName = randomNoun();
-    const randomYesNo = randomBoolean() === true ? YesNoEnum.yes : YesNoEnum.no;
-    return {
-      firstName: firstName,
-      middleName: randomNoun(),
-      lastName: lastName,
-      // Question: why are these strings?
-      birthMonth: randomBirthMonth(),
-      birthDay: randomBirthDay(),
-      birthYear: randomBirthYear(),
-      sameAddress: randomYesNo,
-      // Question: should this be an enum?
-      relationship: randomAdjective(),
-      workInRegion: randomYesNo,
-      householdMemberAddress: randomBoolean
-        ? undefined
-        : { create: addressFactory() },
-      householdMemberWorkAddress: {
-        create: addressFactory(),
-      },
-    };
+export const householdMemberFactorySingle = (
+  index: number,
+): Prisma.HouseholdMemberCreateWithoutApplicationsInput => {
+  const firstName = randomNoun();
+  const lastName = randomNoun();
+  const sameAddress = randomBoolean() === true ? YesNoEnum.yes : YesNoEnum.no;
+  const workInRegion = randomBoolean() === true ? YesNoEnum.yes : YesNoEnum.no;
+
+  const relationshipKeys = Object.values(HouseholdMemberRelationship);
+
+  return {
+    firstName: firstName,
+    middleName: randomNoun(),
+    lastName: lastName,
+    // Question: why are these strings?
+    birthMonth: randomBirthMonth(),
+    birthDay: randomBirthDay(),
+    birthYear: randomBirthYear(),
+    sameAddress,
+    relationship: relationshipKeys[randomInt(relationshipKeys.length)],
+    workInRegion: workInRegion,
+    householdMemberAddress:
+      sameAddress === YesNoEnum.no ? { create: addressFactory() } : undefined,
+    householdMemberWorkAddress:
+      workInRegion === YesNoEnum.yes
+        ? {
+            create: addressFactory(),
+          }
+        : undefined,
+    orderId: index,
```

Comment:
Without this, we were previously unable to view individual household members on an application in partners